README.md

```diff
@@ -25,12 +25,11 @@ See the [live demo](https://onthegomap.github.io/planetiler-demo/) of vector til
 the [OpenStreetMap Americana Project](https://github.com/ZeLonewolf/openstreetmap-americana/).
```

Comment:
Let's add a short blurb somewhere in the readme about pulling the project using submodules so it doesn't trip people up too much. I think we only need to cover the initial pull, and subsequent updates but not making changes or pushing to the submodule.

Reply:
I added a short description. Not sure, where exactly in READme should be located though.

Reply:
Maybe under "Usage" section, something like "planetiler has a submodule dependency, to clone the repo ... and to pull the latest changes ..." ?

Reply:
Moved under Usage section.